### ISSUE TITLE
train and comparison with BSP-Net

### ISSUE THREAD

**Issue by elmiraloo** [edward1997104/Neural-Template#10]
For benchmarking against different datasets and comparing to https://github.com/czq142857/BSP-NET-pytorch/blob/master/main.py if I want to train a model with BSP-net only (without the flow part) I only need to change the loss function a little bit and use the flow decoder and  that should work. Is there any other part that I'm missing?

**Comment by edward1997104** [comment]
In case you want to make a comparison with BSP-Net, I will suggest you to use their official repositories as the hyper-parameters such as the number of planes and convexes are quite different. In our experiments, we compare them using their official implementation.
